node/src/gql/mutation/login-google.ts

```diff
@@ -10,7 +10,9 @@ import { User as UserSchema, Mentor as MentorSchema } from 'models';
 import {
```

Comment:
does mutation/login.ts also need to be updated?

Reply:
no, this route is only called from context. And on every call, it generates a new token.

We will probably stop calling this from the front end. As we have changed the working as follows:
1. on login you get two tokens, i.e short live access token (15 min) and long live refresh token (90 days)
2. as soon as your access token expires, the frontend will automatically get a new access token from the backend using the long-lived refresh token

So we will not need to call login.ts mutation to refresh token on every reload anymore